### PR TITLE
Javascript functions practice

### PR DESCRIPTION
**Pull request by Jc-Leon**

### CODE REVIEW

File: javascript-functions-practice/get-element-at-index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* exported getElementAtIndex */
+function getElementAtIndex (array, index) {
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === array[index] ) {
+      want = array[i]
+    }
+  }
```
Comment (codingcodymiller) [inline javascript-functions-practice/get-element-at-index.js:7]: This should not require a loop

File: javascript-functions-practice/get-last-character.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* exported getLastCharacter */
+function getLastCharacter(string) {
+  var newArray = string.split("");
+  var result;
+  for (let i = 0; i < newArray.length; i++) {
+    if (newArray[newArray.length-1]) {
+      result = newArray[i];
+    }
+  }
```
Comment (codingcodymiller) [inline javascript-functions-practice/get-last-character.js:9]: This should not require a loop. Strings have indexes just like arrays, how can you access the value at the last index of an array or string?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jc, nice start here! Some notes for you:

Also, please include a screenshot with all your tests passing

**Comment by Jc-Leon** [comment]
Screen Shot 2021-07-06 at 11 12 46 AM
Screen Shot 2021-07-06 at 11 12 14 AM
Screen Shot 2021-07-06 at 11 12 26 AM
Screen Shot 2021-07-06 at 11 12 37 AM

File: javascript-functions-practice/get-last-character.js [hunk 3]
```diff
@@ -1,11 +1,4 @@
...
 function getLastCharacter(string) {
-  var newArray = string.split("");
-  var result;
-  for (let i = 0; i < newArray.length; i++) {
-    if (newArray[newArray.length-1]) {
-      result = newArray[i];
-    }
-  }
-  return result;
+  return string.length = string[string.length-1]
```
Comment (codingcodymiller) [inline javascript-functions-practice/get-last-character.js:3]: This should just be `return string[string.length - 1]`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**